src/core/lib/surface/server.h

```diff
@@ -162,7 +174,16 @@ class Server : public ServerInterface,
   // Adds a listener to the server.  When the server starts, it will call
   // the listener's Start() method, and when it shuts down, it will orphan
   // the listener.
-  void AddListener(OrphanablePtr<ListenerInterface> listener);
+  void AddListener(RefCountedPtr<ListenerInterface> listener);
+
+  /// Takes an Endpoint for an established connection, and treats it as if the
+  /// connection had been accepted by the server.
+  ///
+  /// The server must be started before endpoints can be accepted.
+  absl::Status AcceptConnectedEndpoint(
```

Comment:
I think we no longer need this method.  The only reason we needed this before was because we needed to use the lock in the server code, but we reverted that change.  Now I think `PassiveListenerImpl` can directly call the method on the listener.